### PR TITLE
Add UserActivity calls for bubble usage

### PR DESCRIPTION
**Pull request by davidkelly**
This allows us to see both when a user switches on/off the bubble, and when their bubble is activated.

### CODE REVIEW

**Comment by davidkelly** [comment]
TestPlan

A quick smoke test to be sure the bubble still works is sufficient.  I did make sure the backend calls happened and the data appeared, so no need to have metabase access and check that.

**Comment by CainFoool** [comment]
Testing

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/355/HighFidelity-Beta-PR10605-4dae77cd9d382d5d8162f1df5fb7335f4a82d888.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/355/HighFidelity-Beta-PR10605-4dae77cd9d382d5d8162f1df5fb7335f4a82d888.dmg

File: scripts/system/bubble.js [hunk 1]
```diff
@@ -139,10 +140,12 @@
     }
 
     // When the space bubble is toggled...
-    function onBubbleToggled() {
-        var bubbleActive = Users.getIgnoreRadiusEnabled();
-        writeButtonProperties(bubbleActive);
-        if (bubbleActive) {
+    function onBubbleToggled(enabled, doNotLog) {
```
Comment (zfox23) [inline scripts/system/bubble.js]: @davidkelly, what do you think about adding a comment here that explains that `doNotLog` is not included in the C++ function call to `ignoreRadiusEnabledChanged()`, and will be undefined when called from there?

**Comment by CainFoool** [comment]
# Test Passed 

Bubble still works as expected.

File: scripts/system/bubble.js (hunk 1 above)
Comment (davidkelly) [inline scripts/system/bubble.js]: yea -- otherwise this probably is confusing...  Will do

Comment (zfox23) [inline scripts/system/bubble.js]: thank you!

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/362/HighFidelity-Beta-PR10605-2feea16063bd5c958e82773275bf2666e2153490.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/362/HighFidelity-Beta-PR10605-2feea16063bd5c958e82773275bf2666e2153490.dmg

File: libraries/networking/src/UserActivityLoggerScriptingInterface.cpp [hunk 2]
```diff
@@ -71,6 +71,14 @@ void UserActivityLoggerScriptingInterface::makeUserConnection(QString otherID, b
     doLogAction("makeUserConnection", payload);
 }
 
+void UserActivityLoggerScriptingInterface::bubbleToggled(bool newValue) {
+    doLogAction(newValue ? "bubbleOn" : "bubbleOff");
+}
+
+void UserActivityLoggerScriptingInterface::bubbleActivated() {
+    doLogAction("bubbleActivated");
```
Comment (huffman) [inline libraries/networking/src/UserActivityLoggerScriptingInterface.cpp]: Probably too late, but we should be using underscores for action names, not camelcase. `makeUserConnection` is the only action that uses camelCase that I know of.

Comment (davidkelly) [inline libraries/networking/src/UserActivityLoggerScriptingInterface.cpp]: ohhhhh :(   yes indeed.

**Pull request merged**